src/hooks/usePageViews.js

```diff
@@ -102,8 +105,13 @@ const usePageViews = async () => {
       previousLocation.hash === location.hash
     )
       return;
-    trackPageView(location, previousLocation);
-  }, [location, previousLocation]);
+    const fullPath = `${basePath}${location.pathname}`;
+    const fullPreviousPath = previousLocation && `${basePath}${previousLocation.pathname}`;
+    trackPageView(
+      {...location, pathname: fullPath},
+      previousLocation && {...previousLocation, pathname: fullPreviousPath}
+    );
+  }, [basePath, location, previousLocation]);
```

Comment:
Note from discussion: here the path is constructed manually instead of using `window.location` to avoid using globals and because the react router location can have additional useful state.